lib/index.ts

```diff
@@ -1,21 +1,21 @@
+import type { NextcloudEvents } from './Event.ts'
 /**
  * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
-import type { EventBus } from './EventBus'
-import type { EventHandler } from './EventHandler'
-import type { NextcloudEvents } from './Event'
+import type { EventBus } from './EventBus.ts'
+import type { EventHandler } from './EventHandler.ts'
```

Comment:
Could you fix the comment position here?

Reply:
Damn fixed another on and thought I catched all...